.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,56 @@
+name: integration-tests
+
+on:
+  pull_request_review:
+    types: submitted
+
+jobs:
+  build:
+    runs-on: [ubuntu-latest, macos-latest]
```

Comment:
we have decided that we can run this on windows wsl. does this take it into account?

Reply:
You must set up some steps for using WSL, so it will be easier to make a separate file for it.

Reply:
No, it's running on ubuntu or mac, not on wsl. I will add my changes to wsl too. Thanks!

Reply:
Done